Pipfile

```diff
@@ -6,9 +6,10 @@ verify_ssl = true
 [dev-packages]
 pytest = "*"
 tox = "*"
-flake8 = "*"
-black = "*"
+flake8 = "==3.8.4"
```

Comment:
Can you add the following as a comment in the `Pipfile` so people aren't confused: https://github.com/pytest-dev/pytest-html/pull/414/files#r541766102

Reply:
Done.